bin/v-add-sys-web-terminal

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# info: add system web terminal
+# options: NONE
+#
+# example: v-add-sys-web-terminal
+#
+# This function enables the web terminal.
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+if [ "$WEB_TERMINAL" = 'true' ]; then
+	exit
+fi
+
+# Perform verification if read-only mode is enabled
+check_hestia_demo_mode
+
+#----------------------------------------------------------#
+#                       Action                             #
+#----------------------------------------------------------#
+
+# Updating FIREWAL_SYSTEM value
+$BIN/v-change-sys-config-value "WEB_TERMINAL" "true"
```

Comment:
If you write something to sys config make also sure to update syshealth.sh 

https://github.com/hestiacp/hestiacp/blob/537a8ead3f720f27ff37b66180a72e76c73247eb/func/syshealth.sh#L524-L527

And add default value to "false"

Reply:
Will do!